latest/index.bs

```diff
@@ -192,7 +185,10 @@ keys as specified below for discovering certain types of data, especially images
     - Units for "space" axes: 'angstrom', 'attometer', 'centimeter', 'decimeter', 'exameter', 'femtometer', 'foot', 'gigameter', 'hectometer', 'inch', 'kilometer', 'megameter', 'meter', 'micrometer', 'mile', 'millimeter', 'nanometer', 'parsec', 'petameter', 'picometer', 'terameter', 'yard', 'yoctometer', 'yottameter', 'zeptometer', 'zettameter'
     - Units for "time" axes: 'attosecond', 'centisecond', 'day', 'decisecond', 'exasecond', 'femtosecond', 'gigasecond', 'hectosecond', 'hour', 'kilosecond', 'megasecond', 'microsecond', 'millisecond', 'minute', 'nanosecond', 'petasecond', 'picosecond', 'second', 'terasecond', 'yoctosecond', 'yottasecond', 'zeptosecond', 'zettasecond'
 
-If part of [[#multiscale-md]], the length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
+The "axes" are used as part of [[#multiscale-md]]. The length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
+
+The "dimension_names" attribute in the `zarr.json` of the Zarr array of a multiscale level MUS match the names in the "axes" metadata.
```

Comment:
typo `MUS`. Is the `dimension_names` required by the Zarr v3 spec? If not, then maybe we don't need to require it be duplicated here?

Reply:
`dimension_names` is not mandatory in the Zarr spec. However, I think it is useful to have for interoperability because the `dimension_names` lives in the array metadata and `axes` in the group metadata.

Reply:
+1 for making `dimension_names` required for ome-ngff arrays

Reply:
I added `dimension_names` validation into https://github.com/ome/ome-ngff-validator/pull/36